flask_praetorian/base.py

```diff
@@ -484,3 +550,111 @@ def pack_header_for_user(
             **custom_claims
         )
         return {self.header_name: self.header_type + ' ' + token}
+
+    def send_registration_email(self, user=None, template=None,
```

Comment:
I prefer the following style for argument lines exceeding standard width:

```suggestion
    def send_registration_email(
        self, user=None, template=None,
        subject=None, **kwargs,
    )
```

Reply:
submitted